packages/binding-mqtt/src/mqtt-broker-server.ts

```diff
@@ -39,6 +39,18 @@ import { ActionElement, PropertyElement } from "wot-thing-description-types";
 
 const { info, debug, error, warn } = createLoggers("binding-mqtt", "mqtt-broker-server");
 
+enum SegmentLength {
+    "action" = 3,
+    "property" = 4,
+}
+
+enum SegmentIndex {
+    "thingName" = 0,
+    "interactionType" = 1,
+    "interactionName" = 2,
+    "interactionExtension" = 3,
+}
+
```

Comment:
Can you explain your trail of thoughts to use an enum to model topic parsing?

Reply:
For me it was really easy to mix up numbers while coding so I thought it would be easier to use something more explanatory. Since topic segments always have the similar order, I think with enums the code becomes more readable.

Reply:
Personally I would not have used enums but I will definitely not object if you prefer keeping it.
Note: In other bindings we don't use enums for the same type of hierarchy.

Reply:
If it is not a problem I would follow the principles of "less surprise" and "consistency". You can use a prefix or suffix to group semantically `const`. Sorry, for being picky you did a great job already but I think it is worth doing this small refactor.

Reply:
I did the change here: [3952483](https://github.com/eclipse/thingweb.node-wot/pull/976/commits/395248339d74b23bf1f41cbd22c70c01988859fc).